src/scripts/process_us_ca.py

```
@@ -97,82 +130,247 @@ def is_historical_noise(text):
+    category = classify_category(cleaned_full_text, title, country)
+
+    # 섹션별로 분할
+    chunks = split_into_articles(cleaned_full_text)
+    
+    for chunk in chunks:
+        # article_no 정리
+        article_no = chunk['article_no']
+        if article_no:
+            article_no = clean_text(article_no)
+        else:
+            article_no = "General Provisions"
```

Ruff (0.15.6)

[warning] Use ternary operator `article_no = clean_text(article_no) if article_no else "General Provisions"` instead of `if`-`else`-block

Replace `if`-`else`-block with `article_no = clean_text(article_no) if article_no else "General Provisions"`

(SIM108)